src/routes/popup/explore.tsx

```diff
@@ -317,6 +352,22 @@ const CategoryIcon = styled.div`
   color: ${(props) => props.theme.primaryText};
 `;
 
+const FixedHeader = styled.div`
+  display: flex;
+  gap: 1rem;
+  flex-direction: column;
+  position: fixed;
+  width: 100vw;
+  padding: 24px 24px 16px 24px;
```

Comment:
```suggestion
  padding: 24px 24px 16px 24px;
  margin-left: -24px;
```
Fixed Header is left-padded by 24px

Reply:
@7i7o Seems just before committing I removed `left: 0` which caused this issue 😅.  And, it should be fixed now.